src/BlockTypes/AbstractProductGrid.php

```diff
@@ -647,7 +647,7 @@ protected function get_add_to_cart( $product ) {
 			'data-product_id'  => $product->get_id(),
 			'data-product_sku' => $product->get_sku(),
 			'rel'              => 'nofollow',
-			'class'            => 'wp-block-button__link add_to_cart_button',
+			'class'            => 'wp-block-button__link ' . wc_wp_theme_get_element_class_name( 'button' ) . ' add_to_cart_button',
```

Comment:
If we use this function directly, we should update [the `WC requires at least` version to 7.0](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/woocommerce-gutenberg-products-block.php#L12), which is when `wc_wp_theme_get_element_class_name()` will be introduced. Another option would be to add a check around it to make sure it exists before calling it.

Reply:
> Pre-approving, but left a comment about adding a check around the PHP function or raising the required WC version (which we should do anyway in the next release).

Thanks for the review and thanks for the great catch! I added a check around the PHP function. I'm going to merge it!